### PR TITLE
Chore signup phone authentication

### PR DESCRIPTION
**Pull request by raffo1234**
- Urls: `/signup-phone`, `/signup-authentication`
- Expected:
signup-phone
signup-authentication

### CODE REVIEW

File: src/components/SignupAuthenticationPage/SignupAuthenticationPage.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { Link } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import Scrollable from "components/shared/Scrollable";
+import Heading from "components/shared/Heading";
+import Field from "components/shared/Field";
+import AlertMessage from "components/shared/AlertMessage";
+import Button from "components/shared/Button";
+import FormLabel from "components/shared/FormLabel";
+import logotipo from "images/logotipo.png";
```
Comment (alessandrapereyra) [inline src/components/SignupAuthenticationPage/SignupAuthenticationPage.js:11]: Could we rename it to English? 🙏

File: src/components/SignupAuthenticationPage/SignupAuthenticationPage.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import styles from "./SignupAuthenticationPage.module.css";
+
+const SignupAuthenticationSchema = yup.object().shape({
+  code: yup
+    .string()
+    .required("Authentication code is required")
+    .matches(/^(123456)/, "Sorry, that code does not look correct"),
```
Comment (alessandrapereyra) [inline src/components/SignupAuthenticationPage/SignupAuthenticationPage.js:18]: Could you validate numbers only?

File: src/components/SignupAuthenticationPage/SignupAuthenticationPage.js (hunk 1 above)
Comment (raffo1234) [reply src/components/SignupAuthenticationPage/SignupAuthenticationPage.js]: Done

File: src/components/SignupAuthenticationPage/SignupAuthenticationPage.js (hunk 2 above)
Comment (raffo1234) [reply src/components/SignupAuthenticationPage/SignupAuthenticationPage.js]: Done
It is now: `/^\d{1,6}$/`

**Pull request merged**